### ISSUE TITLE
Migrate project to python3.x

### ISSUE THREAD

**Issue by Bastian82** [ronys/pypwsafe#20]
I belive migrating code to python3.x is one of the first thing, that has to be done, before any other improvements can happen.

**Comment by ronys** [comment]
Indeed. I've started this in the 2to3 branch, but abandoned the effort for now, for reasons described in the README.md in that branch.
In any case, the main project (https://github.com/pwsafe/pwsafe) now has a command-line interface (pwsafe-cli), which users of this code might find useful.
